docs/release-notes.md

```diff
@@ -1,7 +1,13 @@
 # Firefox Lockbox for Desktop Release Notes
 
+## 2.1.0-alpha
+
+Baseline release, release notes will be updated from this point on.
```

Comment:
it would be nice to carry on the same format and style from 1.x.y, and highlight this is a major (breaking) shift from the previous releases.

Reply:
I can try to provide that text.

Reply:
I wasn't sure what to include here since we hadn't been tracking features previously, would love to include something though